frontend/packages/operator-lifecycle-manager/src/components/clusterserviceversion.tsx

```diff
@@ -241,11 +241,45 @@ const SubscriptionStatus = ({ muted = false, subscription }) => {
   );
 };
 
+const getLinkForInstallPage = (
+  subscription: SubscriptionKind,
+  namespace: string,
+  obj: ClusterServiceVersionKind | SubscriptionKind,
+) => {
+  const installPageRoute = `/operatorhub/install?pkg=${_.get(
+    subscription,
+    'metadata.name',
+    '',
+  )}&catalog=${_.get(subscription, 'spec.source', '')}&catalogNamespace=${_.get(
+    subscription,
+    'spec.sourceNamespace',
+    '',
+  )}&targetNamespace=${namespace}&csvName=${obj.spec.startingCSV || obj.metadata.name}`;
+  let csvStatus = _.get(obj, 'status.phase', ClusterServiceVersionPhase.CSVPhaseUnknown);
+  if (csvStatus === ClusterServiceVersionPhase.CSVPhaseUnknown) {
+    csvStatus = _.get(obj, 'status.state', ClusterServiceVersionPhase.CSVPhaseUnknown);
+  }
+  return (
+    <a href={installPageRoute} title={'View Install'}>
+      View Install
+    </a>
+  );
+};
+
 const ClusterServiceVersionStatus: React.FC<ClusterServiceVersionStatusProps> = ({
   obj,
   subscription,
+  activeNamespace,
 }) => {
-  const status = obj?.status?.phase;
+  let status = obj?.status?.phase || '';
+  // Handle status when CSV is being copied from openshift-operators->namespace
+  if (
+    status === ClusterServiceVersionPhase.CSVPhaseSucceeded &&
+    obj.metadata.namespace !== activeNamespace
+  ) {
+    status = ClusterServiceVersionPhase.CSVPhasePending;
+  }
```

Comment:
@zherman0 Can you explain what's happening here in a little more detail?

Reply:
We are discussing this now in terms of architecture, but the idea is:  obj here is the combined global CSV objects and namespaced CSV objects.  When installing operators globally, the CSV will not show in the list until it is copied over into the namespace, which can take a long time if there are 200 namespaces for example.  So I now show the CSVs that are being installed globally.  However, the gloabl CSV will show 'Succeeded' long before it is copied into the namespace, so here I check to see if the namespace is the active namepsace or if it is the global namespace.  If it is the global namespace, then the install is not done, we must wait for the copy to occur so I change the state to Pending. Not sure how clear that is, but I may remove this logic as there may be edge cases where it does not work. TBD